doc/tutorials/introduction/documenting_opencv/documentation_tutorial.markdown

```diff
@@ -574,7 +628,7 @@ Write the tutorial {#tutorial_documentation_steps_tutorial}
     <em>"table_of_content_*.markdown"</em> file with the needed table and place new record in it
     similar to existing ones.
     @verbatim
--   @subpage tutorial_windows_visual_studio_image_watch
+-   @subpage_info{tutorial_windows_visual_studio_image_watch}
```

Comment:
But if subpage can still be used, it should be mentioned in the tutorial right ?